databroker/src/broker.rs

```diff
@@ -112,6 +114,10 @@ pub enum Field {
     ActuatorTarget,
     MetadataUnit,
 }
+#[derive(Debug, Clone)]
+pub struct StackVecField {
+    pub svec: SmallVec<[Field; 3]>,
```

Comment:
do we expect 3 to be the "optimal" number because if I understand correctly after 3 elements it will store it anyways in the heap right?

Reply:
for ` pub fields: StackVecField,` this might be the right approach since fields is [current, target, metadata]